src/c++/perf_analyzer/genai-perf/genai_perf/main.py

```diff
@@ -45,11 +45,12 @@ def init_logging() -> None:
     logging.init_logging()
 
 
-def create_artifacts_dirs():
+def create_artifacts_dirs(generate_graphs: bool) -> None:
     if not os.path.exists("artifacts"):
         os.mkdir(f"{DEFAULT_ARTIFACT_DIR}")
         os.mkdir(f"{DEFAULT_ARTIFACT_DIR}/data")
-        os.mkdir(f"{DEFAULT_ARTIFACT_DIR}/images")
+        if generate_graphs:
+            os.mkdir(f"{DEFAULT_ARTIFACT_DIR}/images")
```

Comment:
I think plots is more clear than images
```suggestion
            os.mkdir(f"{DEFAULT_ARTIFACT_DIR}/plots")
```

Reply:
Updated